src/deprecation.h

```diff
@@ -8,7 +8,7 @@
 // Deprecation policy:
 // * Shut down 16 weeks' worth of blocks after the estimated release block height.
 // * A warning is shown during the 2 weeks' worth of blocks prior to shut down.
-static const int APPROX_RELEASE_HEIGHT = 632000;
+static const int APPROX_RELEASE_HEIGHT = 634650;
```

Comment:
This height is not used because of the override below.

This is ok if we allow 2.1.0-1 to reach EoS halt at the same time as 2.1.0. I think that's acceptable; any disagreement?

Reply:
I think it's acceptable too.

Reply:
Ditto.